htdocs/web_portal/views/user/view_user.php

```diff
@@ -72,28 +72,38 @@
             </ul>
         </div>
 
-
+        <?php
+        $isUserAdmin = ($params['user'] && $params['user']->isAdmin());
```

Comment:
Would `$isUserAdmin = $params['user']->isAdmin();` be sufficent here?

Reply:
Yes, Greg. It is sufficient as NULL will be taken care by IF condition. Updated the code.